GaelAlburo/backend-project
# Prepare Reviews API for Containerized Deployment

**Description:**  
This PR prepares the Reviews API for deployment in a Docker container. It includes updates to support containerization, such as adjusting import paths, adding a health check route, and creating a `Dockerfile` for building the container image. Additionally, the `.env` file has been introduced to manage environment variables securely, and the `requirements.txt` file has been relocated for better integration with the Docker build process.  

**Changes:**  
- Created a `Dockerfile` to enable the Reviews API to be built and deployed as a containerized service.
- Added a `/api/v1/reviews/healthcheck` route to verify the health and readiness of the Reviews API container.  
- Updated import paths across the module to ensure compatibility within the containerized environment.  
- Moved `requirements.txt` into the `/reviews_API` directory for seamless dependency installation during Docker image creation.  
- Introduced a `.env` file to centralize environment-specific variables securely.  

**Impact:**  
- Enables the Reviews API to run as a containerized service.  
- Facilitates secure and flexible configuration management with the `.env` file.  
- Adds a health check endpoint to ensure service availability in production environments.  

**Testing:**  
1. Build the Docker image using the `Dockerfile` and verify that the image builds successfully.  
2. Run the container and confirm that the API functions as expected, including the health check route at `/api/v1/reviews/healthcheck`.
